packages/desktop-client/src/components/reports/graphs/NetWorthGraph.tsx

```diff
@@ -38,6 +44,7 @@ function NetWorthGraph({ style, graphData, compact }: NetWorthGraphProps) {
             scale={{ x: 'time', y: 'linear' }}
             theme={chartTheme}
             domainPadding={{ x: 0, y: 10 }}
+            domain={domainMinMax}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ideally we use the same variable name here. Aliasing and renaming things down multiple layers of code leads to difficult to read code.